libs/ballot-interpreter/src/hmpb-rust/timing_marks.rs

```diff
@@ -1691,20 +1693,24 @@ pub fn rect_could_be_timing_mark(geometry: &Geometry, rect: &Rect) -> bool {
     let min_timing_mark_width_multiplier = if is_near_left_or_right_edge {
         0.20
     } else {
-        0.75
+        0.5
     };
     let min_timing_mark_height_multiplier = if is_near_top_or_bottom_edge {
         0.20
     } else {
-        0.75
+        0.5
```

Comment:
You may have already told me this, but can you explain why we need relax these width/height thresholds for rects that aren't near the edges? I know that we have the added filter of the template score that kicks in after this, but are there rects that would have been filtered out by the 0.75 that do end up matching the template well?

Reply:
Yes, some have been filtered out for being too skinny/short. In the TRR corpus most of the problem was the opposite, but in our existing fixtures and previous backups I found this to be helpful.